app/views/members/_search.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= form_tag(members_path, method: :get) do %>
+  <%= text_field_tag :query, params[:query] %>
```

Comment:
From a security standpoint, all user input is inherently untrusted. There are many types of _injection attacks_ _(SQLi, FI, XXE, etc.)_ against web applications, and XSS _(Cross-Site Scripting)_ is basically a script injection attack. Every place user input is handled should use some form of input sanitization _(don't forget to ensure the proper sanitization scope!)_.

From the [Rails Security Guide](http://guides.rubyonrails.org/security.html) section on [XSS](http://guides.rubyonrails.org/security.html#cross-site-scripting-xss):

> The most common entry points are message posts, user comments, and guest books, but project titles, document names **and search result pages** have also been vulnerable - just about everywhere where the user can input data.

In this case, the `text_field_tag` usage should look something like:

```erb
 <%= text_field_tag :query, sanitize(params[:query]) %>
```

> so assumed the model sanitize was enough

While I did encourage usage of `sanitize_sql_like` in the model `search_result` method, that is not the same value being reflected back to the search form in this instance. 

Regarind Rails Model security, I definitely recommend also checking out the [SQL Injection](http://guides.rubyonrails.org/security.html#sql-injection) and [Unsafe Query Generation](http://guides.rubyonrails.org/security.html#unsafe-query-generation) sections of the Rails security guide while you're in there.